vocs/sidebar.ts

```diff
@@ -46,10 +46,12 @@ export const sidebar: Sidebar = [
     ]
    },
    {
-    text: 'Chain Interactions', 
+    text: 'Contract Interactions', 
```

Comment:
same as the previous PR, when i click on `contract interactions` title, it collapses sidebar below- best to just move this content into an `introduction` section and leave the title as is